### PR TITLE
Bugfix and Testing

### PR DESCRIPTION
**Pull request by phetter**
On the Home screen, if one clicks the ellipsis there is a "View on Neoscan" option. Before this was not going to the correct location.

This has been fixed to operate dynamically when switching networks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "classnames": "^2.1.3",
     "eslint-cli": "^1.1.1",
     "history": "^4.7.2",
+    "lodash": "^4.17.11",
```
Comment (DalderupMaurice) [inline package.json]: Finally hehe 🙈 
Fredrik was against this as we only needed a few functions and didn't want to bloat the codebase, but I think it's way more beneficiary to utilize this as we could leverage lodash in multiple places to reduce code and improve readability.

**Comment by phetter** [comment]
The tests cleared locally, but not here. I'm going to postpone worrying about the tests until we get through these changes. 

If you see any code issues let me know. I'll merge at end of week or after review.

**Comment by phetter** [comment]
Adding internal note as well to fix SwitchAccount test

**Comment by phetter** [comment]
Sorry about the commit stream. I was running push instead of test script; will avoid in future.

I resolved the switchaccount test for now. It's a kludge that uses setTimeout, but I have the start of a promise system in the works. Will resolve asap.

**Pull request merged**